Add librarian-sdr package

goodbadwolf opened the pull request:
This adds librarian-sdr buildroot package, which handles SDR demodulator installation for librarian.

goodbadwolf committed:
Add librarian-sdr package

foxbunny commented on python-librarian-sdr/Config.in:
```diff
@@ -0,0 +1,8 @@
+config BR2_PACKAGE_PYTHON_LIBRARIAN_SDR
+	bool "python-librarian-sdr"
+	depends on BR2_PACKAGE_PYTHON || BR2_PACKAGE_PYTHON3
+	select BR2_PACKAGE_PYTHON_LIBRARIAN
```
Packages that have "PYTHON_LIBRARIAN" in the name, cannot select "PYTHON_LIBRARIAN" pkg. Some weirdness from buildroot.

Commit:
Fix dependency issue

Due to dependency on librarian being implicitly declared by package naming
convetnion, we need to get rid of select line to prevent circular dependency.

Signed-off-by: Branko Vukelic <branko@outernet.is>
